src/main/java/frc/robot/commands/autos/OneFiveBumpReturn.java

```diff
@@ -103,7 +109,24 @@ public OneFiveBumpReturn(
                       intake.setDesiredClamped(false);
                       intake.stopIntakingGamePiece();
                     })),
-        new DriveDistance(drive, NORM_SPEED_INTAKING, -X_METERS_TO_CONE, 0.0, red),
-        drive.followTrajectoryCommand(secondTraj, false));
+        // new DriveDistance(drive, NORM_SPEED_I NTAKING, -X_METERS_TO_CONE, 0.0, red),
+        new InstantCommand(
+            () -> scoringLocationUtil.setScoreCol(red ? ScoreCol.LEFT : ScoreCol.RIGHT)),
+        drive.followTrajectoryCommand(secondTraj, false),
+        new InstantCommand(drive::setNoMove, drive),
+        new LookForTag(tagLimelight, drive, lights).withTimeout(0.05),
+        new SwerveFollowerWrapper(drive)
+            .finallyDo(
+                (boolean interrupted) -> {
+                  System.out.println("Driving to tag interrupted? " + interrupted);
+                }),
+        drive.stopDrivingCommand(),
+        new ConditionalCommand(
+            // Score the game piece
+            new SequentialCommandGroup(
+                new WaitCommand(0.02), new ScoreThisGamePiece(fast, lift, lights)),
+            // Do nothing
+            new InstantCommand(),
+            () -> TRY_TO_SCORE));
```

Comment:
Where does this variable get set (aside from initialization)?

Reply:
Nowhere, it's basically a flag so we can easily switch whether we want this to try to score or not. I didn't want to drop the score code